src/math/p_erff.c

```diff
@@ -0,0 +1,25 @@
+#include <pal.h>
+
+/**
+ *
+ * Calculate error function.
+ * 
+ *
+ * @param a     Pointer to input vector
+ *
+ * @param c     Pointer to output vector
+ *
+ * @param n     Size of 'a' and 'c' vector.
+ *
+ * @return      None
+ *
+ */
+#include <math.h>
+void p_erff_f32(const float *a, float *c, int n)
```

Comment:
In my opinion, name can be changed to `p_erf_f32`.
In `math.h` it's named `erff` since it operates on `float`. In pal types are specified as a suffix (eg. `_u64`, `_f32`), so the additional `f` can be omitted.